tests/bsim/bluetooth/host/att/mtu_update/test_scripts/run_test.sh

```diff
@@ -5,13 +5,14 @@
 set -eu
 source ${ZEPHYR_BASE}/tests/bsim/sh_common.source
 
-simulation_id="$(guess_test_long_name)"
```

Comment:
can't comment on a deleted file. Could you revert `_compile.sh`? You can rename it to `compile.sh` to align with @alwa-nordic 's https://github.com/zephyrproject-rtos/zephyr/pull/76956